### DISCUSSION TITLE
GFW

### DISCUSSION THREAD

**Discussion by alaskasquirrel** [DNSCrypt/dnscrypt-proxy#1905] [Potential issues]
ODoH is partially interfered by GFW in China.

GFW filtering rules：‘alidns-doh’，‘dnspod-doh’，‘tuna-doh-ipv4’ 

dnscrypt-proxy.toml：
> disabled_server_names = ['alidns-doh','dnspod-doh','tuna-doh-ipv4']
> forwarding_rules = 'forwarding-rules.txt'

**Comment by jedisct1** [comment]
ODoH is very easy to block.

Ideally, DoH and oDoH servers should be hosted on the same domains as large websites that content blockers would never block. But this is not the case yet, so you have to keep trying to find servers that are not blocked.
